pm-commotion-mac2ip

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+#
+# Script written for PittMesh by Jason Khanlar
+#
+
+# Check for --listall, otherwise proceed
+
+if [ "$*" = "--listall" ]; then
```

Comment:
I think I'd like to see the body of this `if` be a separate function, or maybe even a separate script.